### ISSUE TITLE
Can't load stream.el any more

### ISSUE THREAD

**Issue by ghost** [NicolasPetton/stream#2]
Hello,

I rebuilt Emacs from the sources right now, and can't require stream any more.

To reproduce: In emacs -Q, load seq.el, and thunk.el, and then try to load stream.el.  I get the following output with an error:

```
Load stream.el? (y or n) y
Loading /home/micha/.emacs.d/elpa/stream-2.0.1/stream.el (source)...
Compiler-macro error for cl-generic-make-generalizer: (wrong-number-of-arguments (5 . 5) 4) [2 times]
Load error for /home/micha/.emacs.d/elpa/stream-2.0.1/stream.el:
(wrong-number-of-arguments (4 . 4) 3)
```

TIA.

**Comment by NicolasPetton** [comment]
Thanks for reporting it! It should be fixed with commit https://github.com/NicolasPetton/stream/commit/94b9358b2fc8f8c9c5ca97b812352938b6c68ece.

**Comment by ghost** [comment]
Nicolas Petton notifications@github.com writes:

> Thanks for reporting it! It should be fixed with commit 94b9358.

Confirmed.  Thanks!
